src/components/Client/ClientSettings.js

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import axios from 'axios';
+import { appRoutes } from '../../utils/configs';
+import { ClientMenu } from '../../utils/ClientMenu';
+
+const ClientSettings = () => {
+  const history = useHistory();
+  const menuItems = ClientMenu();
+
+  const [fields, setFields] = useState({
+    phone: '',
+    oldPassword: '',
+    newPassword: '',
+    cPassword: '',
+    name: '',
+    alert: '',
+  });
+
+  const onClick = (event) => {
+    event.preventDefault();
+    if (fields.newPassword.length < 6) {
+      setFields({ ...fields, alert: 'password length should be at least 6 characters, try again!', newPassword: '' });
+    } else if (fields.newPassword === fields.oldPassword) {
+      setFields({
+        ...fields, alert: 'new passwords should differ from old password, try again!', newPassword: '', cPassword: '',
+      });
+    } else if (fields.oldPassword.length < 6) {
+      setFields({ ...fields, alert: 'old password length should be at least 6 characters, try again!', oldPassword: '' });
+    } else if (fields.newPassword !== fields.cPassword) {
+      setFields({
+        ...fields, alert: 'incorrect password, try again!', newPassword: '', cPassword: '',
+      });
+    } else if (!fields.name) {
+      setFields({ ...fields, alert: 'username is required!' });
+    } else if (!fields.phone) {
+      setFields({ ...fields, alert: 'phone is required!' });
+    } else {
+      axios.post('/client/settings', {
+        phone: fields.phone,
+        username: fields.name,
+        password: fields.newPassword,
+      })
+        .then((response) => response)
+        .catch((error) => error);
+      history.push(appRoutes.client);
+    }
+  };
+
+  return (
```

Comment:
Big components make debugging harder, try to avoid them

Reply:
This component's not work yet, It's from first PR Sevak. 
If it does not interfere, we will replace it later.